### PR TITLE
[cs] Place CS rules in config file

### PR DESCRIPTION
**Pull request by Maks3w**
Supersede and close https://github.com/zendframework/zend-diactoros/pull/28/

### CODE REVIEW

**Comment by localheinz** [comment]
@Maks3w 

:+1:

File: composer.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
   },
   "require-dev": {
     "phpunit/PHPUnit": "~4.6",
-    "squizlabs/php_codesniffer": "~2.0"
+    "squizlabs/php_codesniffer": "^2.3.1"
```
Comment (weierophinney) [inline composer.json]: Just curious, why this specific version and up? What changed specifically with 2.3.1 that is needed?

Comment (Maks3w) [inline composer.json]: long array rule not exists before 2.3.1

Comment (Maks3w) [inline composer.json]: Yea they didn't follow semantic versioning about to raise feature number when new sniffs are added

Comment (weierophinney) [inline composer.json]: Okay, cool; was just curious. :)

**Pull request merged**